modules/core/multihop/proof_gen.go

```diff
@@ -18,6 +19,7 @@ import (
 // Endpoint is stateless from caller's perspective.
 type Endpoint interface {
 	ChainID() string
+	GetChainHeight() int64
```

Comment:
is this for debugging only? I'd recommend to keep the interface as lean as possible

Reply:
Yes I'll remove it after I resolve the client update issue. Sorry didn't mean to add it to the PR